### DISCUSSION TITLE
Is it possible to use a dynamic basePath with local routing?

### DISCUSSION THREAD

**Discussion by mdegrees** [vercel/next.js#23584] [Help] [answered]
My use case (multi-tenancy) entails using localization at the tenant path. My domain looks like ``https://foo.com/${tenantname}``. When using ``<Link href="/page" locale="fr">``,  I want the destination to be ``https://foo.com/${tenantname}/fr/page``. What I get instead is ``https://foo.com/fr/page``.  Is this kind of routing possible with a custom config?

thank you

**Comment by aberba** [comment, marked as answer]
Why not `https://foo.com/fr/${tanantName}/page` ?

**Reply by mdegrees** [reply]
That's what I ended up using, thank you
